protocol/testing/containertest/containertest.sh

```diff
@@ -154,27 +153,22 @@ setup_cosmovisor() {
 	done
 }
 
-download_preupgrade_binary() {
+copy_preupgrade_binary_arch() {
```

Comment:
Can start using v3.0.0-rc0 binaries (building right now and will be published soon for testnet).

Reply:
Can we just swap them for the 3.0.0 binaries once they are qualified. I don't suspect that this swap will cause many issues and then we will only need to do it the one time.